### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.compute.compute.v2015_06_15.models.api_error_base.ApiErrorBase
  fullName: azure.mgmt.compute.compute.v2015_06_15.models.api_error_base
  langs:
  - python
  module: azure.mgmt.compute.compute.v2015_06_15.models.api_error_base
  name: api_error_base
  source:
    id: api_error_base
    path: azure-mgmt-compute\azure\mgmt\compute\compute\v2015_06_15\models\api_error_base.py
    remote:
      branch: master
      path: azure-mgmt-compute\azure\mgmt\compute\compute\v2015_06_15\models\api_error_base.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.compute.compute.v2015_06_15.models.api_error_base
references:
- fullName: azure.mgmt.compute.compute.v2015_06_15.models.api_error_base.ApiErrorBase
  isExternal: false
  name: ApiErrorBase
  parent: azure.mgmt.compute.compute.v2015_06_15.models.api_error_base
  uid: azure.mgmt.compute.compute.v2015_06_15.models.api_error_base.ApiErrorBase
